testing/trino-tests/src/test/java/io/trino/execution/TestConnectorEventListener.java

```diff
@@ -31,7 +32,9 @@
 @Test(singleThreaded = true)
 public class TestConnectorEventListener
 {
-    private final EventsCollector generatedEvents = new EventsCollector();
+    private final EventsCollector generatedEvents = new EventsCollector(EventFilters.builder()
```

Comment:
nit: should we do the same in `TestCompletedEventWarnings` for event types not tested?